moja-kuhinjica-frontend/src/components/buttons/style.js

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components'
+import arrow from '../../../public/static/assets/images/arrow.svg'
+import redArrow from '../../../public/static/assets/images/redArrow.svg'
+
+export const WhiteButton = styled('button')`
+    /* position: absolute; */
```

Comment:
If you wanted this to be commented out than you are wrong this is working.  Please remove this `/*` from style if not needed

Reply:
Will do.